api.js

```diff
@@ -247,17 +248,29 @@ class Api extends EventEmitter {
 			});
 		});
 
+		let interrupted = false;
+
 		return this._computeForkExecArgs(files)
 			.then(argvList => {
 				execArgvList = argvList;
 			})
 			.return(files)
 			.map((file, index) => {
-				return new Promise(resolve => {
+				return new Promise((resolve, reject) => {
 					const forkArgs = execArgvList[index];
+					if (interrupted) {
+						const relFile = path.relative('.', file);
+						return reject(new AvaError(`File ${relFile} was not run.`));
```

Comment:
AVA prints this as an uncaught exception, which isn't ideal. Do you think this even needs to be printed, or can these test files be skipped silently?